system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_unstuck.rs

```diff
@@ -0,0 +1,101 @@
+
+
+pub struct UnstuckSnowbridge;
+
+impl OnRuntimeUpgrade for UnstuckSnowbridge {
```

Comment:
You also need to add it to the list of migrations ([example](https://github.com/polkadot-fellows/runtimes/blob/c2886b940944ba96c87b6e07954e0d945b9a67bd/system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs#L151))

Reply:
Thanks for the reminder! https://github.com/polkadot-fellows/runtimes/pull/313/commits/b13f7a97601e91759ebd8c5aabfb3a334f03b418